scalafix-core/shared/src/main/scala/scalafix/internal/rule/RuleCtxImpl.scala

```diff
@@ -96,14 +96,15 @@ case class RuleCtxImpl(tree: Tree, config: ScalafixConfig) extends RuleCtx {
   def replaceSymbol(fromSymbol: Symbol.Global, toSymbol: Symbol.Global)(
       implicit index: SemanticdbIndex): Patch =
     TreePatch.ReplaceSymbol(fromSymbol, toSymbol)
-  def replaceSymbols(toReplace: (String, String)*)(
```

Comment:
We will have to keep this option for a while to preserve compatibility. I am fine with using the dummy implicit trick as a workaround throughout v0.5.